docs/working_with_data_sources.md

```diff
@@ -0,0 +1,203 @@
+---
+id: working_with_data_sources
+author: GoodData
+sidebar_label: Working with Data Sources
+title: Working with Data Sources
+---
+
+Creating a Data Source
+-------
+
+You can create a Snowflake data source programmatically:
+
+```ruby
+# encoding: utf-8
+
+require 'gooddata'
+
+snowflake_ds_model = {
+          'dataSource' => {
+              'name' => 'My Data Source',
+              'alias' => 'ds_alias',
+              'prefix' => 'ds_prefix_',
+              'connectionInfo' => {
+                  'snowflake' => {
+                      'url' => 'jdbc:snowflake://test',
+                      'authentication' => {
+                          'basic' => {
+                              'userName' => 'testing_user',
+                              'password' => '***'
+                          }
+                      },
+                      'database' => 'db_test',
+                      'schema' => 'schema_test',
+                      'warehouse' => 'wh_test'
+                  }
+              }
+          }
+      }
+client = GoodData.connect
+snowflake_ds = client.create(GoodData::DataSource, snowflake_ds_model)
+snowflake_ds.save
+
+# after the data source is created then you can start working with it
+puts snowflake_ds.name # => 'My Data Source'
```

Comment:
Working data source should be deploying a process using the data source, e.g deploy LDM brick, using Snowflake data source as source input, deploy schedule with param reference to the data source.

Reply:
Done update code